pytest/tests/sanity/state_sync_massive.py

```diff
@@ -0,0 +1,129 @@
+# Survive massive state sync
+#
+# Create 3 nodes, 1 validator and 2 observers tracking the single shard 0.
```

Comment:
what's the point of having two observers? It seems better for the purpose of testing to only have one peer to sync from.

Reply:
We are only syncing from 1, but the other is necessary for node0 to start validating, it won't start without connected peers.

Reply:
> We are only syncing from 1, but the other is necessary for node0 to start validating, it won't start without connected peers.

Seems like a configuration issue. We can change the config to reduce min_num_peers to 0 for node0.